Cvičení 1/LightSystem.cpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include <glm/glm.hpp>
+#include "LightSystem.h"
+#include "Shader.h"
+#include "SimpleLight.h"
+
+void LightSystem::add(const SimpleLight& light)
+{
+    //podle n�jak�ho p�rka na StackOverflow je "emplace_back" lep�� ne� "push_back" ale kdo v�. Kr�sy 3. str�nky po google search
```

Comment:
`push_back()` totiž kompletně překopíruje a alokuje novej vektor, `emplace_back()` to alokuje jinde a akorát dohodí pointer k tomu stávajícímu... má to svoje pro i proti.

Já všude používám `push_back` cuz idc